src/components/i18n.js

```diff
@@ -8,10 +8,18 @@ i18n
   .use(LanguageDetector)
   .use(initReactI18next)
   .init({
-    // ns: ["translations"],
-    // defaultNS: "translations",
+    ns: ["translations"],
+    defaultNS: "translations",
     debug: true,
     fallbackLng: "en",
+    resources: {
+      en: {
+        translations: require("../locales/en/translation.json"),
```

Comment:
This seems to be duplicating line 9 in `gatsby-node.js`. Either delete `onPostBuild()` in `gatsby-node.js` (preferred) or delete lines 15-22 in `i18n.js`.